go.mod

```diff
@@ -51,6 +51,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk v1.7.0
 	github.com/hashicorp/vault v1.3.0 // indirect
 	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c // indirect
+	github.com/jcpowermac/terraform-provider-vsphereprivate v0.0.0-20200314174956-1c9da6dfe728 // indirect
```

Comment:
I need to fix this

Reply:
I think we can keep the entire code in the installer repo like the azureprivatedns

Reply:
@abhinavdahiya yeah its all under plugins/vsphereprivate just need to figure out how to fix it.